src/main/java/com/google/sps/data/EmojiSelection.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+public final class EmojiSelection {
+
+  private static String[] emoji1 = {"👿","🖕","😾","😡","😠","😧","💔"};
```

Comment:
I think it's important to keep this appropriate. Removing the first two emojis would be great.

Reply:
You're right, I went ahead an removed it